tendrl/commons/message.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, priority, publisher, payload, job_id=None,
         else:
             self.caller = caller
         self.priority = priority
+        if not publisher:
+            publisher = NS.publisher_id
```

Comment:
@anmolbabu it can be may be, according to me each function must not worry about all the errors , some information should be compulsory for the function to work else  everything will have to be placed under try and except, but I am not sure if this is the right way to code, and it's a small change 

@r0h4n if possible could you give your opinion , I can make the amends as per suggestions